spec/features/idv/steps/in_person/verify_info_spec.rb

```diff
@@ -11,6 +11,7 @@
 
   before do
     allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+    allow(IdentityConfig.store).to receive(:in_person_state_id_controller_enabled).and_return(false)
     allow_any_instance_of(ApplicationController).to receive(:analytics).and_return(fake_analytics)
     allow(user).to receive(:enrollment).
       and_return(enrollment)
```

Comment:
I am not sure the best practice here...

1. I think maybe we should enable the flag here to show new State ID controller is working with the Verify Info page. This would require setting `in_person_state_id_controller_enabled` to true  and updating all instance of `complete_state_id_step(user)` to `complete_state_id_controller(user)` as you previously have done.

2. Copy/paste tests and then have an enabled and disabled context block of tests to show both. As I write this- I realize we are not doing this everyone (to show both) in all of the above tests.

I am leaning towards 1 because of lift and because we are moving towards enabling the flag.  It would be nice to hear what others think.

Reply:
Ah you are right, I didn't treat this test the same as others where I've switched the tests to use the controller helper instead of the step one. I can definitely take the second approach and make sure both versions are being tested here. And I'll look at doing the same for the other feature specs.

Reply:
I updated this spec to include both scenarios, the FSM one and the state id controller one.

However in looking at others, in particular `in_person_spec.rb`, I'm realizing that there are a few helper methods that call the state id step, and also more helper methods that use _those_ methods that use the state id step. So since there's some nested method calls, that means I'll have to make an "FSM" version for a handful of helper methods to call on the old state id FSM "step", only to delete them later. I understand the benefit here is that it does allow us to continue to test the feature flag "false" state until we remove it but I'm not sure it's worth making a number of separate feature test helper methods.

Curious what you and others think @gina-yamada !

Reply:
If we agree that it's too much new code just to test the old feature flag, I will still be sure to keep the changes to this file and others consistent with testing the "true"/controller version. It was a good catch that I only flipped this one to expect "false" originally.